resources/javascript/functions-and-methods.js

```diff
@@ -12,6 +13,18 @@ class FunctionsAndMethods {
     static #testFunction3() {    // complexity +1
         return 2;
     }
+    get latest() {
+        return 3;
+    }
+    set latest(age) {
+        this.#age = age;
+    }
+    *more(){
+        let index = 0;
+        while (true) {
+            yield index++;
+    }
+}
```

Comment:
should add complexity counting to match the whole file
also add space after *more()
;)
```suggestion
    get latest() {    // complexity +1
        return 3;
    }
    set latest(age) {    // complexity +1
        this.#age = age;
    }
    *more() {    // complexity +1
        let index = 0;
        while (true) {    // complexity +1
            yield index++;
    }
}
```